fargate-services/2017-emergency-response-api.yaml

```diff
@@ -182,3 +234,40 @@ Resources:
                                 "Resource": "*"
                         }]
                     }
+                - PolicyName: ssm-access
+                  PolicyDocument: 
+                    {
+                        "Version": "2012-10-17",
+                        "Statement": [
+                            {
+                                "Effect": "Allow",
+                                "Action": [
+                                    "ssm:DescribeParameters"
+                                ],
+                                "Resource": "*"
+                            },
+                            {
+                                "Sid": "Stmt1482841904000",
+                                "Effect": "Allow",
+                                "Action": [
+                                    "ssm:GetParameters"
+                                ],
+                                "Resource": [
+                                    "arn:aws:ssm:us-west-2:845828040396:parameter/staging/2018/*",
+                                    "arn:aws:ssm:us-west-2:845828040396:parameter/production/2018/*",
+                                    "arn:aws:ssm:us-west-2:845828040396:parameter/staging/2019/*",
+                                    "arn:aws:ssm:us-west-2:845828040396:parameter/production/2019/*"
+                                ]
+                            },
+                            {
+                                "Sid": "Stmt1482841948000",
+                                "Effect": "Allow",
+                                "Action": [
+                                    "kms:Decrypt"
+                                ],
+                                "Resource": [
+                                    "arn:aws:kms:us-west-2:845828040396:key/0280a59b-d8f5-44e0-8b51-80aec2f27275"
+                                ]
+                            }
+                        ]
+                    }
```

Comment:
Is there an open issue to refactor these TaskRoles to be provided to the stack rather than duplicated across every service?

Reply:
That was on my list of to-do's, and there is now:
https://github.com/hackoregon/civic-devops/issues/249